src/pages/managerDashboard.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Link from 'next/link';
+import Image from 'next/image';
+import NavBar from '@/components/Header/NavBar';
+import FooterBar from '@/components/common/FooterBar';
+import DashboardListing from '@/components/singledashboardlisting/DashboardListing';
+// import styles from '../styles/managerDashboard.module.css';
+
+function managerDashboard () {
+  return (
+    <div>
+        <NavBar />
+        <main className='bg-white container mx-auto p-10'>
+            <div className='bg-primary-lite my-10 rounded-md flex justify-between sm-text-center'>
+                <div className='flex flex-col gap-5 p-10  lg:text-left'>
+                    <h1 className='text-5xl text-blue-800'>Welcome Back, Emmanuel!</h1>
+                    <p className='text-2xl'>You have 8 new members!</p>
```

Comment:
@istoga So our app does not actually have the ability to track how many people join the communities that we list. I think for now, we should remove this sentence.

For the future (perhaps a future team!), we could decide to implement a click-tracking feature. We track the unique clicks that a listing receives between the manager's logins. When they login, we can then say something like "Your listings received interest from 8 people since your last login"

But for now let us remove this sentence. 🙏🏾